main.js

```diff
@@ -308,7 +308,7 @@ function share(inputGame){
         }
     }
 
-    shareStr = "Вордел "+[d.getUTCFullYear(),d.getUTCMonth(),d.getUTCDate()].join('-')+'\n'+shareStr
+    shareStr = "Вордел "+[d.getUTCFullYear(),d.getUTCMonth()+1,d.getUTCDate()].join('-')+'\n'+shareStr
```

Comment:
@svvvst the `getmonth()` function returns the month as a number between 0 and 11. So I've incremented the value by one to match your date format.

Reply:
Thanks for your help! Didn't have a chance to get around to this!